int32 id
int32 type
geometry_msgs/Point position
